src/lib/autocomplete/BUILD.bazel

```diff
@@ -21,6 +21,11 @@ ng_module(
   tsconfig = ":tsconfig-build.json",
 )
 
+# TODO(jelbourn): replace this w/ sass_library when it supports acting like a filegroup
```

Comment:
we could make a simple PR to rules_sass to get this process started, at least test the waters to see if anyone is maintaining it, if they offer to hand over the keys?

Reply:
I'll add it to my general "Making sass better" task list